src/test/java/com/wikia/webdriver/testcases/mobilewikitests/TrackingOptInModalTests.java

```diff
@@ -0,0 +1,91 @@
+package com.wikia.webdriver.testcases.mobilewikitests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.dataprovider.TrackingOptInDataProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.TrackingOptInModal;
+
+import org.testng.annotations.Test;
+
+
+@Test(groups = {"mobile-wiki-tracking-opt-in"})
+@InBrowser(
+    browser = Browser.CHROME,
+    emulator = Emulator.GOOGLE_NEXUS_5
+)
+public class TrackingOptInModalTests extends NewTestTemplate {
+
+  @Execute(asUser = User.ANONYMOUS, trackingOptIn = false)
+  @Test(groups = {"mobile-wiki-tracking-opt-in"},
+      dataProviderClass = TrackingOptInDataProvider.class,
+      dataProvider = "GDPRcountries"
+  )
+  public void testModalVisibilityForAnon(String continent, String country, boolean shouldGetModal) {
+    TrackingOptInModal.setGeoCookie(driver, continent, country);
+    new ArticlePage().open();
+
+    PageObjectLogging.logInfo("Geo cookie: ", driver.manage().getCookieNamed("Geo").getValue());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Geo cookie: " 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)